crates/bevy_wgpu/Cargo.toml

```diff
@@ -29,7 +29,7 @@ bevy_winit = { path = "../bevy_winit", optional = true, version = "0.5.0" }
 bevy_utils = { path = "../bevy_utils", version = "0.5.0" }
 
 # other
-wgpu = "0.9"
+wgpu = { version = "0.10.0", features = ["spirv"] }
```

Comment:
There's already 0.10.1. I didn't check what was in it but maybe worth checking...

Reply:
Looks like they just excluded some unnecessary resources from the crate: https://github.com/gfx-rs/wgpu/commit/12c82b9b9e9987a86cf968353b1311c63616ca53#diff-056ecc5c9d4f624f6ddb10662b92cb15bbfcc32d4f3e662e6cfcc17ec7777580.

Technically we want that version to be the oldest supported version right, so we should just leave it `0.10.0` in that case? To make our dependency as compatible as possible with other crates that depend on the same crate?

Reply:
yup, 0.10.1 is semver compatible with 0.10.0 so it will be picked by default by cargo